hoomd/hpmc/pytest/test_faceted_ellipsoid.py

```diff
@@ -1,31 +1,314 @@
 import hoomd
 import hoomd.hpmc
 import hoomd.hpmc._hpmc as hpmc
+import pytest
+import copy
+
+args_1 = {"normals": [(0, 0, 1),
```

Comment:
The typical way in pytest to have module level variables for test like this is to use fixtures. Could we consider them here? They look more explicit in the test definitions.